docs/persistence/postgres.md

```diff
@@ -67,6 +67,60 @@ Default 7 backups will be kept. More info [here](https://cloud.google.com/sql/do
 
 The backups can be found in the [Google Cloud SQL instance](https://cloud.google.com/sql) dashboard.
 
+### Personal database access
+Databases should always be accessed using a personal account, and the access should ideally be temporary.
+Here's how:
+
+#### Prerequisites
+When the instance is created, we need to grant the IAM users access to the "public" schema.
+This can either be done by using the default application database user during database creation/migration with scripts (e.g. flyway), or as a one-time setup by using the default postgres user:
+
+In order to use the postgres user, you have to set a password first:
+```bash
+gcloud sql users set-password postgres --instance=<INSTANCE_NAME> --prompt-for-password --project <PROJECT_ID>
```

Comment:
Har man tilgang til å gjøre dette rett ut av boksen? Burde kanskje stå noe om hvilken tilgang man trenger for å kunne gjøre dette?

Reply:
Det samme gjelder de to andre `gcloud projects`-kommandoene.

Reply:
Godt poeng. Kan kanskje skippes hvis dette blir en del default rollen utviklerene tildeles

Reply:
Ville tro at man gikk for en så minimal rolle som mulig, som bare ga de muligheten til å gi seg selv andre roller. Å koble seg til en postgres-database er jo ikke noe de trenger tilgang til hele tiden.

Reply:
Ser nå at det bare er en `gcloud projects`-kommando, og den er jeg enig med deg om burde fungere ut av boksen, da den setter iam-rolle. `add-iam-policy-binding`